transcom/mymove
# Mb 14327 cant delete orders once submitted

## [Jira ticket](https://dp3.atlassian.net/browse/MB-14327)

Per Prakit’s comment in the ticket users can now no longer delete their orders once submitted. So, if their move is in `DRAFT` status, they can delete a previously uploaded order. Once they submit and the move goes to `SUBMITTED` status, they can no longer delete orders. They can still upload new or updated orders.


### How to test

1. Go through move creation and upload your orders
2. Before submitting the move, you will see the `Delete` button and are able to delete the orders file
3. After submitting the move, you will not see the `Delete` button as you should not be able to delete orders after submitting a move.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
